src/profile.ts

```diff
@@ -10,11 +10,54 @@ async function addUserInfo (uok: UsernameOrKaid): Promise<void> {
 	const User = await getUserData(uok);
 	const Scratchpads = await getJSON(`${userEndpoint}/scratchpads?kaid=${User.kaid}&limit=1000`, {
 		scratchpads: [{
+			url: 1,
 			sumVotesIncremented: 1,
 			spinoffCount: 1
 		}]
 	}) as Scratchpads;
 
+	const first = Scratchpads.scratchpads[0];
+	if (first) {
+		const url = window.location.origin +
+			"/api/internal/show_scratchpad?scratchpad_id=" +
+			first.url.split("/")[5];
+
+		Promise.all([
+			querySelectorPromise(
+				".user-info.clearfix",
+				10, 500
+			),
+			querySelectorPromise(
+				"._o77ufew",
+				10, 500
+			),
+			querySelectorPromise(
+				"._19lfck2n",
```

Comment:
Seems like we talked about it in #175. I think that issue was an Aphrodite update which changed every generated class. And when I fixed that, I tired to remove our reliance on them. But the few that we still have haven't broken in a while.
It looks like the hashes are constant as long as KA doesn't change the styling on the element. So since they haven't touched the CSS of the CS section in a couple years, everything's the same.

Reply:
How time flies ... that makes sense. I guess we're kind of stuck unless we use some base element with a fixed identifier and move through the tree like `.wrapper > div > div` which might be less stable, probably slower to query, and more characters. 😱